### DISCUSSION TITLE
iCub RealSense holder CAD file

### DISCUSSION THREAD

**Discussion by patrickshas** [robotology/community#687] [Q&A] [answered]
Hello all,
We need to slightly adjust the angle of the RealSense platform on the holder.
Wondering if there is a CAD model available for the holder (rather than STL).
Thanks in advance

**Comment by pattacini** [comment, marked as answer]
Hi @patrickshas 

You can find the STEP file to import into your CAD at https://icub-tech-iit.github.io/documentation/upgrade_kits/realsense_holder/support/#print-the-realsense-holder-yourself.

If this is not enough, you can find the CREO source files at https://github.com/icub-tech-iit/cad-mechanics-public/tree/master/projects/gadgets/ig_027_RealSense_head_holder.

CC'ing @Mick3Lozzo who can be more precise than me in this respect.

**Comment by patrickshas** [comment]
@pattacini Thank you.
Much appreciated.
